acceptance/bundle/deploy/secret-scope/permissions/output.txt

```diff
@@ -0,0 +1,10 @@
+
+>>> [CLI] bundle deploy
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-secret-scope-with-permissions/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> jq -s -c .[] | select(.path=="/api/2.0/secrets/acls/put") | .body out.requests.txt
+{"permission":"MANAGE","principal":"admins","scope":"test-secrets-permissions"}
+{"permission":"READ","principal":"users","scope":"test-secrets-permissions"}
```

Comment:
Do you know why the current user is missing from the output here?

Reply:
that's because dabs silently filters out all permissions for the current user name in https://github.com/databricks/cli/blob/47d329cffdfc40b7793ba153a749d5f5b2140523/bundle/config/mutator/resourcemutator/filter_current_user.go#L13-L15

Reply:
Can you confirm this actually leads to the desired behavior here?

If I recall correctly, the integration test user is also a member of the "admins" group, so secret r/w will still be possible in the integration test, but may not be if you use a non-admin user to provision it.

Reply:
Secret scope api always give the caller the MANAGE permission if initial_manage_principal is omitted (and in dabs we always omit it). All the other permissions that we specify here give additional groups permission and they do not affect the permission that the caller has.

This is also shown by the `bundle/deploy/secret-scope` test in this PR

Reply:
Thanks -- I mistakenly thought this was the output of the above, and thought the current user had their ACL stripped entirely, but this is the set of PUT requests from the client into the server.

```
>>> [CLI] secrets list-acls my-secrets-[UUID]
{"permission":"MANAGE","principal":"[USERNAME]"}
{"permission":"READ","principal":"users"}
{"permission":"WRITE","principal":"admins"}
```